### PR TITLE
[SELS-TASK] BE-Store-Word-API

### PR DESCRIPTION
**Pull request by ragg09-sunasterisk**
### Asana Link
https://app.asana.com/0/1203341829928748/1203341829928691/f

### Commands
```
cd backend
php artisan serve
cd frontend
npm start
```
### Pre-conditions
-first option field will always be the correct translation

### Expected Output
- [X] be able to store word and its options

### Notes
- in form, first option field must be the correct translation, I am using array for better BE logic,  I will make this obvious in FE soon
- I include GET WORDS API since I dont have this task and I need it for my next tasks

### Screenshots
image
image

### CODE REVIEW

File: be/app/Http/Controllers/WordController.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                $options = new Word_choice();
+                $options->word_id = $words->id;
+                $options->word = $request->option[$i];
+                $options->remark = $remark;
+                $options->save();
+            }
+        }
+
+
+
```
Comment (escbooster12-dev) [inline be/app/Http/Controllers/WordController.php]: - remove multiple newlines

File: fe/src/pages/BufferPage.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from "react";
```
Comment (escbooster12-dev) [inline fe/src/pages/BufferPage.jsx]: - is this the `AuthMiddleware`?

File: fe/src/pages/word/CreateWord.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
+import React, { Fragment } from "react";
+
+function CreateWord(props) {
+  const handleGetCategory = (e) => {
+    console.log(props.id);
+  };
+  return (
```
Comment (escbooster12-dev) [inline fe/src/pages/word/CreateWord.jsx]: - add newlione
```suggestion
  };

  return (
```

File: fe/src/pages/BufferPage.jsx (hunk 2 above)
Comment (ragg09-sunasterisk) [reply fe/src/pages/BufferPage.jsx]: @escbooster12-dev  yes sir, Ill remove it

**Review by escbooster12-dev — APPROVED** [review]

**Pull request merged**